test/spec/modules/sovrnBidAdapter_spec.js

```diff
@@ -632,55 +622,18 @@ describe('sovrnBidAdapter', function() {
             }
           }
         },
-        'headers': {}
+        headers: {}
       }
-      const returnStatement = spec.getUserSyncs(pixelEnabledOptions, [...serverResponse, resp2]);
-      expect(returnStatement.length).to.equal(4);
+
+      const returnStatement = spec.getUserSyncs(pixelEnabledOptions, [...serverResponse, resp2])
+
+      expect(returnStatement.length).to.equal(4)
       expect(returnStatement).to.deep.include.members([
         { type: 'image', url: 'http://idprovider1.com' },
         { type: 'image', url: 'http://idprovider2.com' },
         { type: 'image', url: 'http://idprovider3.com' },
         { type: 'image', url: 'http://idprovider4.com' }
-      ]);
-    });
-  });
-
-  describe('prebid 3 upgrade', function() {
```

Comment:
Why is this no longer needed?

Reply:
I agree, this test seems to still be necessary.